File: app/src/main/java/in/kashewdevelopers/todo/provider_classes/PrefManager.java
package in.kashewdevelopers.todo.provider_classes;

import android.content.Context;
import android.content.SharedPreferences;

public class PrefManager {

    public static void setSortType(Context context, int sortType) {
        SharedPreferences sharedPref = context.getSharedPreferences(Constants.SORT, Context.MODE_PRIVATE);

        SharedPreferences.Editor editor = sharedPref.edit();

        editor.putInt(Constants.SORT_CODE, sortType);
        editor.apply();
    }

    public static int getSortType(Context context) {
        SharedPreferences sharedPref = context.getSharedPreferences(Constants.SORT, Context.MODE_PRIVATE);
        return sharedPref.getInt(Constants.SORT_CODE, Constants.SORT_OLD_EARLY);
    }

}

File: app/src/main/java/in/kashewdevelopers/todo/provider_classes/PrefManager.kt
package `in`.kashewdevelopers.todo.provider_classes

import android.content.Context

class PrefManager {

    companion object {
        fun setSortType(context: Context, sortType: Int) {
            val sharedPref = context.getSharedPreferences(Constants.SORT, Context.MODE_PRIVATE)

            val editor = sharedPref.edit()

            editor.putInt(Constants.SORT_CODE, sortType)
            editor.apply()
        }

        fun getSortType(context: Context): Int {
            val sharedPref = context.getSharedPreferences(Constants.SORT, Context.MODE_PRIVATE)
            return sharedPref.getInt(Constants.SORT_CODE, Constants.SORT_OLD_EARLY)
        }
    }

}
